webapi/src/main/scala/org/knora/webapi/routing/InvalidTokenCache.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright © 2021 - 2024 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.routing
+
+import zio.ZIO
+import zio.ZLayer
+
+import org.knora.webapi.slice.infrastructure.CacheManager
+import org.knora.webapi.slice.infrastructure.EhCache
+
+final case class InvalidTokenCache(cache: EhCache[String, String]) {
+  def put(token: String): Unit         = cache.put(token, "")
+  def contains(token: String): Boolean = cache.get(token).isDefined
+}
```

Comment:
If this interface is for tokens only, why is it not stored in a special way to avoid clashes or why is the cache not used directly?

Reply:
Ah, it's a new cache, it wraps an anonymous cache created when instantiating the layer. Fine.

Reply:
In general I think a cache should most probably only be used by a single component and not be shared because sharing a cache makes it hard to track down bug should they occur.

Unfortunately, this "cache" is shared between two components. The `Authenticator` writing into and the the `JwtService` reading from it. I have modeled it as a dedicated component so it is clear for what it is used and can be shared easily. 

The approach of "directly" using the cache required each component using the cache knowledge like the cache name and its key and value types. If one of them did not match or changed you would have runtime problems and hard to track down errors.